### PR TITLE
Add TAC Minutes & Supporting Docs For 2024-04-04

### PR DESCRIPTION
**Pull request by Riaankl**
This PR includes the meeting artifacts from the 2024-04-04 TAC meeting.

@bensternthal

### CODE REVIEW

**Review by bensternthal — APPROVED** [review]
Looks good.

File: TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+#### Other Attendees
+
+* Bob Reiter (Booz Allen Hamilton)
+* Bokdeuk Jeong (Samsung)
+* Drasko Draskovic
+* Gregory Starkey (Booz Allen Hamilton)
+* Henry Wang (Meta)
+* Henk Birkholz
+* Jorg Rodel (Suse)
+*  Kevin Hui (Meta)
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md:58]:
```suggestion
* Kevin Hui (Meta)
```
typo / whitespace

File: TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+* Manu Foutaine (Hushmesh)
+* Matthieu Legre (Cysec)
+* Nathaniel McCallium (Red Hat)
+* Nick Vidal
+* Nicklas Maeding
+* Pawan Khandavilli
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md:64]:
```suggestion
* Pawan Khandavilli (Microsoft)
```

File: TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+* Rachel Wan (IBM)
+* Richard Zak
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md:66]:
```suggestion
* Richard Zak (Enarx Maintainer; Booz Allen Hamilton)
```

File: TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+Mike Bursell was unable to attend and therefore there was not Executive Director update. 
+
+## New Outreach Sig Demos
+
+Pawan Khandavilli (PK) from Microsoft briefly introduced themselves. PK stated that the Outreach team identified demos as a missing part of their events and marketing strategy. PK stated that the team is collecting demos and suggested that videos might be the best approach. PK asked the TAC to submit demos to him and Outreach so they can be potentially showcased at future events. 
+
+
+## Enarx Annual Review
+
+Richard Zak (RZ), Gregory Starkey (GS), and Nick Vidal (NV) presented the [Enarx annual review](./VirTee%20Demo.pdf). Discussion ensued.
```
Comment (fritzalder) [inline TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md:116]:
```suggestion
Richard Zak (RZ), Gregory Starkey (GS), and Nick Vidal (NV) presented the [Enarx annual review](./Enarx%20Review%202024.pdf). Discussion ensued.
```

Comment (Riaankl) [reply TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md]: Good catch, thank you.

File: TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+* Gregory Starkey (Booz Allen Hamilton)
+* Henry Wang (Meta)
+* Henk Birkholz
+* Jorg Rodel (Suse)
+* Kevin Hui (Meta)
+* Manu Foutaine (Hushmesh)
+* Matthieu Legre (Cysec)
+* Nathaniel McCallium (Red Hat)
+* Nick Vidal
+* Nicklas Maeding
```
[the remark this answers is not in the document]
Comment (Riaankl) [reply TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md]: Updated, thank you.

File: TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+   " + " indicates attending
+
+#### Other Attendees
+
+* Bob Reiter (Booz Allen Hamilton)
+* Bokdeuk Jeong (Samsung)
+* Drasko Draskovic
+* Gregory Starkey (Booz Allen Hamilton)
+* Henry Wang (Meta)
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md:55]: moved to above
```suggestion
```

File: TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+* Henk Birkholz
+* Jorg Rodel (Suse)
+* Kevin Hui (Meta)
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md:58]: moved to above
```suggestion
```

File: TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+* Jorg Rodel (Suse)
+* Manu Foutaine (Hushmesh)
+* Matthieu Legre (Cysec)
+* Nathaniel McCallium (Red Hat)
+* Nick Vidal
+* Nicklas Maeding (IBM)
+* Pawan Khandavilli (Microsoft)
+* Rachel Wan (IBM)
+* Richard Zak (Enarx Maintainer; Booz Allen Hamilton)
+* Tyler Fanelli (Red Hat)
```
Comment (thomas-fossati) [inline TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md:65]:
```suggestion
* Thomas Fossati (Linaro)
* Tyler Fanelli (Red Hat)
```

**Review by dcmiddle — APPROVED** [review]

File: TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+* Bokdeuk Jeong (Samsung)
+* Drasko Draskovic
+* Gregory Starkey (Booz Allen Hamilton)
+* Henk Birkholz
+* Jorg Rodel (Suse)
+* Manu Foutaine (Hushmesh)
+* Matthieu Legre (Cysec)
+* Nathaniel McCallium (Red Hat)
+* Nick Vidal
+* Nicklas Maeding (IBM)
```
Comment (nmaeding-ibmde) [inline TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md:61]: Nicolas , please :)

Comment (Riaankl) [reply TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md:61]: Updated in #235

**Pull request merged**